### ISSUE TITLE
.java files did not compile in codecrunch even though they compiled and worked in vim

### ISSUE THREAD

**Issue by gracefour11** [nus-cs2030/2021-s2#14]
I just realised that I had these error: **gcc: error trying to exec 'ecj1': execvp: No such file or directory**  in Recitation 1. I basically only submitted .java files and not .jsh files, and I'm wondering if these could be the reason for this error. I created a Main.java to test my codes and they compiled and executed properly in vim on my laptop. Anyone has thoughts on this?

**Comment by Yung-Jer** [comment]
I have the same problem too :(  I'm also using a driver class to test my codes. However, I copied the body of the driver class and pasted in notepad to generate .jsh file and handed in together with the driver class. Does anyone have the same problem?

**Comment by Buwoo** [comment]
Yep same, have this error. Submitted both the .java and .jsh files, but was not too sure if I formatted the .jsh file correctly. Tested in jshell and worked fine, both files were submitted but not sure why I got the aforementioned error message in the Test Output too

**Comment by yowyuxuan** [comment]
I've faced the same issue too! However, I just submit my .java file. Tried to make it work but failed.

**Comment by lxin42140** [comment]
From code crunch: `Unlike your lab assignments, your submissions will not be graded, but still we would be reading through them so that we can pick some outstanding ones for discussion later.`

The submissions for recs are not graded so the grade doesn't matter, it is for participation only. The TAs will just be looking through the submissions manually for rect tomorrow :) Kudos for submitting your solutions !

**Comment by hwkchia** [comment]
I've set the recitation tasks on CodeCrunch to hide output.  Hopefully, you don't get bothered by these from now on.
